<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentInfo</class>
 <widget class="QMainWindow" name="StudentInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StudentInfo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSignIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>133</height>
     </rect>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelUsername">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditUsername"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditPassword"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonLogin">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>183</width>
         <height>19</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
